lib/room.rb

```diff
@@ -0,0 +1,78 @@
+#Room Class
```

Comment:
This class isn't doing much other than providing an `id`.  So... maybe just use an array of numbers for the rooms.